--- sbms_config.py.original	2020-07-07 14:28:39.000000000 -0400
+++ sbms_config.py.fixed	2022-01-21 18:33:53.541971824 -0500
@@ -55,7 +55,7 @@
 	ofdirname = '%s/JANA' % env['variant_dir']
 	ofdir = '%s' % env.Dir(ofdirname)
 	ofname = '%s/jana_config.h' % ofdir
-	print 'sbms : Making jana_config.h in %s' % ofdir
+	print('sbms : Making jana_config.h in %s' % ofdir)
 	
 	# ROOT
 	HAVE_ROOT = 0
@@ -91,13 +91,13 @@
 	env['HAVE_CCDB']   = HAVE_CCDB
 
 	# If showing build, print config. results
-	if(env['SHOWBUILD']>0):
-		print '--- Configuration results ----'
-		print '    HAVE_ROOT = %d' % HAVE_ROOT
-		print '  HAVE_XERCES = %d' % HAVE_XERCES
-		print '      XERCES3 = %d' % XERCES3
-		print '    HAVE_CCDB = %d' % HAVE_CCDB
-		print '------------------------------'
+	if(int(env['SHOWBUILD'])>0):
+		print('--- Configuration results ----')
+		print('    HAVE_ROOT = %d' % HAVE_ROOT)
+		print('  HAVE_XERCES = %d' % HAVE_XERCES)
+		print('      XERCES3 = %d' % XERCES3)
+		print('    HAVE_CCDB = %d' % HAVE_CCDB)
+		print('------------------------------')
 		
 	str = ''
 	
@@ -147,7 +147,7 @@
 	ofdir = '%s' % env.Dir(ofdirname)
 	ofname = '%s/jana-config' % ofdir
 
-	print 'sbms : Making jana-config in %s' % ofdir
+	print('sbms : Making jana-config in %s' % ofdir)
 	
 	# ROOT
 	HAVE_ROOT = 0
