--- sbms_config.py.original	2020-06-12 18:03:40.925031806 -0400
+++ sbms_config.py.patched	2020-06-12 18:32:14.980284799 -0400
@@ -55,7 +55,7 @@
 	ofdirname = '#src/.%s/JANA' % env['OSNAME']
 	ofdir = '%s' % env.Dir(ofdirname)
 	ofname = '%s/jana_config.h' % ofdir
-	print 'sbms : Making jana_config.h in %s' % ofdir
+	print('sbms : Making jana_config.h in %s' % ofdir)
 	
 	# ROOT
 	HAVE_ROOT = 0
@@ -91,13 +91,13 @@
 	env['HAVE_CCDB']   = HAVE_CCDB
 
 	# If showing build, print config. results
-	if(env['SHOWBUILD']>0):
-		print '--- Configuration results ----'
-		print '    HAVE_ROOT = %d' % HAVE_ROOT
-		print '  HAVE_XERCES = %d' % HAVE_XERCES
-		print '      XERCES3 = %d' % XERCES3
-		print '    HAVE_CCDB = %d' % HAVE_CCDB
-		print '------------------------------'
+	if(int(env['SHOWBUILD'])>0):
+		print('--- Configuration results ----')
+		print('    HAVE_ROOT = %d' % HAVE_ROOT)
+		print('  HAVE_XERCES = %d' % HAVE_XERCES)
+		print('      XERCES3 = %d' % XERCES3)
+		print('    HAVE_CCDB = %d' % HAVE_CCDB)
+		print('------------------------------')
 		
 	str = ''
 	
@@ -143,7 +143,7 @@
 	ofdirname = '#%s/bin' % env['OSNAME']
 	ofdir = '%s' % env.Dir(ofdirname)
 	ofname = '%s/jana-config' % ofdir
-	print 'sbms : Making jana-config in %s' % ofdir
+	print('sbms : Making jana-config in %s' % ofdir)
 
 	JANA_INSTALL_DIR = '%s' % env.Dir("#/%s" % env['OSNAME']).abspath
 	
