--- sbms.py.original	2020-03-27 11:38:47.910624018 -0400
+++ sbms.py.patched	2020-03-27 11:56:28.324675578 -0400
@@ -516,13 +516,18 @@
 ##################################
 def AddCCDB(env):
 	ccdb_home = os.getenv('CCDB_HOME', 'none')
+	sqlite_home = os.getenv('SQLITE_HOME', 'none')
 	if(ccdb_home != 'none'):
 		CCDB_CPPPATH = "%s/include" % (ccdb_home)
 		CCDB_LIBPATH = "%s/lib" % (ccdb_home)
 		CCDB_LIBS = "ccdb"
+		SQLITE_LIBPATH = "%s/lib" % (sqlite_home)
+		SQLITE_LIBS = "sqlite3"
 		env.AppendUnique(CPPPATH = [CCDB_CPPPATH])
 		env.AppendUnique(LIBPATH = [CCDB_LIBPATH])
 		env.AppendUnique(LIBS    = [CCDB_LIBS])
+		env.AppendUnique(LIBS    = [SQLITE_LIBS])
+		env.AppendUnique(LIBS    = ['fmt'])
 
 
 ##################################
