--- Imake.cf	2020-05-20 13:56:38.851099400 -0400
+++ Imake.cf.patched	2020-05-20 13:54:10.132940000 -0400
@@ -393,6 +393,30 @@
 #elif __GNUC__ == 3
 #undef __GNUC__
 #define GCC3
+#elif __GNUC__ == 5
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 6
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 7
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 8
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 9
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 10
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 11
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 12
+#undef __GNUC__
+#define GCC4
 #else
 /*  old linux compiler suite (default)  */
 #undef __GNUC__
