--- Imake.cf	2006-06-15 04:59:55.000000000 -0400
+++ Imake.cf.new	2017-07-24 21:40:41.890237500 -0400
@@ -393,6 +393,21 @@
 #elif __GNUC__ == 3
 #undef __GNUC__
 #define GCC3
+#elif __GNUC__ == 5
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 6
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 7
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 8
+#undef __GNUC__
+#define GCC4
+#elif __GNUC__ == 9
+#undef __GNUC__
+#define GCC4
 #else
 /*  old linux compiler suite (default)  */
 #undef __GNUC__
