--- Install_cernlib	2016-12-08 11:27:33.874398406 -0500
+++ Install_cernlib.new	2016-12-08 11:27:04.114326012 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 echo "===================="
 echo "CERNLIB installation"
@@ -9,10 +9,12 @@
 export CERN=$PWD
 
 # Get the original cernlib sources + corrections
+# build_scripts: do Install_cernlib_src in advance of this script so
+# that code can be patched
 
-echo "installing cernlib sources"
+#echo "installing cernlib sources"
 
-./Install_cernlib_src
+#./Install_cernlib_src
 
 
 # Define the cernlib version
@@ -31,6 +33,8 @@
 FC=" "
 [ "$GCCVSN" = "3" ]&&FC=g77
 [ "$GCCVSN" = "4" ]&&FC=gfortran
+[ "$GCCVSN" = "5" ]&&FC=gfortran
+[ "$GCCVSN" = "6" ]&&FC=gfortran
 if [ "$GCCVSN" = " " ]; then
   echo " "
   echo "====================================="
