--- Install_cernlib	2015-03-27 10:00:28.286564347 -0400
+++ Install_cernlib.patched.new	2015-03-27 09:59:55.379728099 -0400
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 echo "===================="
 echo "CERNLIB installation"
@@ -14,6 +14,11 @@
 
 ./Install_cernlib_src
 
+# patch kstring.h
+
+pushd 2005/src/packlib/kuip/kuip
+patch < $BUILD_SCRIPTS/patches/cernlib/kstring.h.patch
+popd
 
 # Define the cernlib version
 
