--- Install_cernlib	2010-07-31 17:16:59.000000000 -0400
+++ Install_cernlib.patched	2020-05-19 08:25:47.521537100 -0400
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 echo "===================="
 echo "CERNLIB installation"
@@ -9,10 +9,12 @@
 export CERN=$PWD
 
 # Get the original cernlib sources + corrections
+# build_scripts: do Install_cernlib_src in advance of this script so
+# that code can be patched
 
-echo "installing cernlib sources"
+#echo "installing cernlib sources"
 
-./Install_cernlib_src
+#./Install_cernlib_src
 
 
 # Define the cernlib version
@@ -27,10 +29,18 @@
       a = 0
       end
 EOF
-GCCVSN=`cpp -dM comptest.F | grep __VERSION__ | cut -d" " -f3 | cut -c 2`
+GCCVSN=`cpp -dM comptest.F | grep __VERSION__ | cut -d\" -f2 | cut -d. -f1`
 FC=" "
 [ "$GCCVSN" = "3" ]&&FC=g77
 [ "$GCCVSN" = "4" ]&&FC=gfortran
+[ "$GCCVSN" = "5" ]&&FC=gfortran
+[ "$GCCVSN" = "6" ]&&FC=gfortran
+[ "$GCCVSN" = "7" ]&&FC=gfortran
+[ "$GCCVSN" = "8" ]&&FC=gfortran
+[ "$GCCVSN" = "9" ]&&FC=gfortran
+[ "$GCCVSN" = "10" ]&&FC=gfortran
+[ "$GCCVSN" = "11" ]&&FC=gfortran
+[ "$GCCVSN" = "12" ]&&FC=gfortran
 if [ "$GCCVSN" = " " ]; then
   echo " "
   echo "====================================="
