--- ROOTConfig.cmake.in.original	2021-10-07 19:36:49.697025913 -0400
+++ ROOTConfig.cmake.in.patched	2021-10-07 19:37:21.418745872 -0400
@@ -67,7 +67,7 @@
 
 #----------------------------------------------------------------------------
 # Include RootMacros.cmake to get ROOT's CMake macros
-include("${_thisdir}/RootMacros.cmake")
+include("${_thisdir}/modules/RootMacros.cmake")
 
 #----------------------------------------------------------------------------
 # Include the file listing all the imported targets and options
