--- Diracxx_original/CMakeLists.txt	2024-01-02 13:52:53.000000000 -0500
+++ Diracxx_patched/CMakeLists.txt	2024-01-02 15:43:53.671650115 -0500
@@ -5,18 +5,8 @@ project(Diracxx VERSION 1.1 DESCRIPTION
 configure_file(DiracxxConfig.h.in DiracxxConfig.h)
 
 # specify the C++ standard
-if(CMAKE_CXX17_STANDARD_COMPILE_OPTION STREQUAL "-std=c++17")
-  message("compile c++ code with c++17 support")
-  set(CMAKE_CXX_STANDARD 17)
-elseif(CMAKE_CXX14_STANDARD_COMPILE_OPTION STREQUAL "-std=c++14")
-  message("compile c++ code with c++14 support")
-  set(CMAKE_CXX_STANDARD 14)
-elseif(CMAKE_CXX11_STANDARD_COMPILE_OPTION STREQUAL "-std=c++11")
-  message("compile c++ code with c++11 support")
-  set(CMAKE_CXX_STANDARD 11)
-else(CMAKE_CXX17_STANDARD_COMPILE_OPTION STREQUAL "-std=c++17")
-  message("compile c++ code with compiler default std")
-endif(CMAKE_CXX17_STANDARD_COMPILE_OPTION STREQUAL "-std=c++17")
+message("compile c++ code with c++11 support")
+set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 # require python to be installed
